src/charm.py

```diff
@@ -21,8 +26,9 @@ def __init__(self, *args):
         self._name = self.model.app.name
         self._namespace = self.model.name
         self._manager_service = "manager"
+        self._src_dir = os.path.dirname(__file__)
```

Comment:
```suggestion
        self._src_dir = Path(__file__).parent
```

We're using Pathlib elsewhere, so... :)

Reply:
Ah shoot, that again haha! I forgot :D Will change it.